### PR TITLE
OSDOCS-7590: Adding metrics dashboard about info

### PR DESCRIPTION
**Pull request by DCChadwick**
Version(s): 4.14

Issue: [OSDOCS-7590](https://issues.redhat.com//browse/OSDOCS-7590) & [OSDOCS-6934](https://issues.redhat.com//browse/OSDOCS-6934)

Link to docs preview: https://65727--docspreview.netlify.app/openshift-enterprise/latest/nodes/nodes-dashboard-using

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/nodes-dashboard-using-about.adoc [hunk 1]
```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-dashboard-using.adoc
+
+:_content-type: CONCEPT
+[id="nodes-dashboard-using-about_{context}"]
+= About the node metrics dashboard
+
+The node metrics dashboard enables administrative and support team members to monitor metrics related to pod scaling, including scaling limits used to diagnose and troubleshoot scaling issues. Particularly, you can use the visual analytics displayed via the dashboard to monitor workload distributions across nodes. Insights gained from these analytics help you determine the health of your CRI-O and Kubelet system components as well as identify potential sources of excessive or inbalanced memory consumption and system instability.
```
Comment (harche) [inline modules/nodes-dashboard-using-about.adoc:9]:
```suggestion
The node metrics dashboard enables administrative and support team members to monitor metrics related to pod scaling, including scaling limits used to diagnose and troubleshoot scaling issues. Particularly, you can use the visual analytics displayed via the dashboard to monitor workload distributions across nodes. Insights gained from these analytics help you determine the health of your CRI-O and Kubelet system components as well as identify potential sources of excessive or inbalanced resource consumption and system instability.
```

**Comment by mwerner2113** [comment]
@harche is this ready for QE?

**Comment by DCChadwick** [comment]
@harche I've made some additions to this PR. If you are okay with these changes, I will tag the PR for QE review.

**Comment by DCChadwick** [comment]
@sunilcio This is ready for QE review.

**Comment by DCChadwick** [comment]
@jfrancoa Thanks for your review. I've made the updates you suggested.

**Review by jfrancoa — APPROVED** [review]
lgtm

File: modules/nodes-dashboard-using-accessing.adoc [hunk 2]
```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-dashboard-using.adoc
+
+:_content-type: PROCEDURE
+[id="nodes-dashboard-using-accessing_{context}"]
+= Accessing the node metrics dashboard
+
+You can access the node metrics dashboard from the Administrator perspective.
```
Comment (adellape) [inline modules/nodes-dashboard-using-accessing.adoc:9]:
```suggestion
You can access the node metrics dashboard from the *Administrator* perspective.
```

File: modules/nodes-dashboard-using-accessing.adoc [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+.Procedure
+
+. Expand the *Observe* menu option and select *Dashboards*.
+. Under the *Dashboard* filter, select *Node cluster*.
+
+[NOTE]
+====
+If no data appears in the visualizations under the Critical category, no critical anomalies were detected. The dashboard is working as intended.
```
Comment (adellape) [inline modules/nodes-dashboard-using-accessing.adoc:18]:
```suggestion
If no data appears in the visualizations under the *Critical* category, no critical anomalies were detected. The dashboard is working as intended.
```

File: modules/nodes-dashboard-using-identify.adoc [hunk 4]
```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-dashboard-using.adoc
+
+:_content-type: CONCEPT
+[id="nodes-dashboard-using-identify_{context}"]
+= Identify metrics for indicating optimal node resource usage
+
+The node metrics dashboard is organized into four categories: Critical, Outliers, Average Durations, and Number of Operations. The metrics in the Critical category help you indicate optimal node resource usage. These metrics include:
```
Comment (adellape) [inline modules/nodes-dashboard-using-identify.adoc:9]: Markup all of the "categories" with `*` for bold, if they're GUI labels.

File: modules/nodes-dashboard-using-identify-critical-top3.adoc [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+The *Top 3 containers with the most OOM kills in the last day* query fetches details regarding the top three containers that have experienced the most Out-Of-Memory (OOM) kills in the previous day. 
+
+.Example default query
+----
+topk(3, sum(increase(container_runtime_crio_containers_oom_count_total[1d])) by (name))
+----
+
+OOM kills force the system to terminate some processes due to low memory. Frequent OOM kills can hinder the functionality of the node and even the entire Kubernetes ecosystem. Containers experiencing frequent OOM kills might be consuming more memory than they should, which causes system instability. 
+
+Use this metric to identify containers that are experiencing frequent OOM kills and investigate why these containers are consuming an excessive amount of memory. Adjust the resource allocation if necessary and consider resizing the containers based on their memory usage. You can also review the metrics under the *Outliers*, *Average durations* and *Number of operations* categories to gain further insights into the health and stability of your nodes.
```
Comment (adellape) [inline modules/nodes-dashboard-using-identify-critical-top3.adoc:18]:
```suggestion
Use this metric to identify containers that are experiencing frequent OOM kills and investigate why these containers are consuming an excessive amount of memory. Adjust the resource allocation if necessary and consider resizing the containers based on their memory usage. You can also review the metrics under the *Outliers*, *Average durations*, and *Number of operations* categories to gain further insights into the health and stability of your nodes.
```

^ Looks like there are 5 total instances of the above list (`*Outliers*, *Average durations* and *Number of operations* categories`) in this PR that are missing a comma, so apply throughout.

File: modules/nodes-dashboard-using-identify-critical-cpu.adoc [hunk 6]
```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-dashboard-using.adoc
+
+:_content-type: CONCEPT
+[id="nodes-dashboard-using-identify-critical-cpu.adoc"]
+= Nodes with System Reserved CPU Utilization > 80%
+
+The *Nodes with system reserved CPU utilization > 80%* query identifies nodes where the syste-reserved CPU utilization is more than 80%. The query focuses on the system-reserved capacity to calculate the rate of CPU usage in the last 5 minutes and compares that to the CPU resources available on the nodes. If the ratio exceeds 80%, the node's result will be displayed in the metric.
```
Comment (adellape) [inline modules/nodes-dashboard-using-identify-critical-cpu.adoc:9]: A typo and a future tense that could be avoided:
```suggestion
The *Nodes with system reserved CPU utilization > 80%* query identifies nodes where the system-reserved CPU utilization is more than 80%. The query focuses on the system-reserved capacity to calculate the rate of CPU usage in the last 5 minutes and compares that to the CPU resources available on the nodes. If the ratio exceeds 80%, the node's result is displayed in the metric.
```

File: modules/nodes-dashboard-using-identify-critical-memory-kubelet.adoc [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+
+.Example default query
+----
+sum by (node) (container_memory_rss{id="/system.slice/kubelet.service"}) / sum by (node) (kube_node_status_capacity{resource="memory"} - kube_node_status_allocatable{resource="memory"}) * 100 >= 50
+----
+
+This query helps you identify any possible memory pressure situations in your nodes that could affect the stability and efficiency of node operations. Kubelet memory utilization that consistently exceeds 50% of the system reserved memory, indicate that the system reserved settings are not configured properly and that there is a high risk of the node becoming unstable. 
+
+If this metric is highlighted, review your configuration policy and consider adjusting the system reserved settings or the resource limits settigns for the Kubelet. Additionally, if your Kubelet memory utilization consistently exceeds half of your total reserved system memory, examine metrics under the *Outliers*, *Average durations* and *Number of operations* categories to gain further insights for more precise diagnostics.
```
Comment (adellape) [inline modules/nodes-dashboard-using-identify-critical-memory-kubelet.adoc:18]:
```suggestion
If this metric is highlighted, review your configuration policy and consider adjusting the system reserved settings or the resource limits settings for the Kubelet. Additionally, if your Kubelet memory utilization consistently exceeds half of your total reserved system memory, examine metrics under the *Outliers*, *Average durations* and *Number of operations* categories to gain further insights for more precise diagnostics.
```

File: modules/nodes-dashboard-using-identify-critical-memory-crio.adoc [hunk 8]
```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-dashboard-using.adoc
+
+:_content-type: CONCEPT
+[id="nodes-dashboard-using-identify-critical-memory-crio.adoc"]
+= Nodes with CRI-O system reserved memory utilization > 50%
+
+The *Nodes with CRI-O system reserved memory utilization > 50%* query calculates all nodes where the percentage of used memory reserved for the CRI-O system is greater than or equal to 50%. In this case, memory usage is defined by the Resident Set Size (RSS), which is the portion of the CRI-O system's memory held in RAM.
```
Comment (adellape) [inline modules/nodes-dashboard-using-identify-critical-memory-crio.adoc:9]: Earlier you had "resident set size (RSS)" not capitalized. I don't think it needs it (seems a general computing term), so suggest lowercasing here as well:
```suggestion
The *Nodes with CRI-O system reserved memory utilization > 50%* query calculates all nodes where the percentage of used memory reserved for the CRI-O system is greater than or equal to 50%. In this case, memory usage is defined by the resident set size (RSS), which is the portion of the CRI-O system's memory held in RAM.
```

File: modules/nodes-dashboard-using-identify-critical-cpu-kubelet.adoc [hunk 9]
```diff
@@ -0,0 +1,18 @@
...
+
+.Example default query
+----
+sum by (node) (rate(container_cpu_usage_seconds_total{id="/system.slice/kubelet.service"}[5m]) * 100) / sum by (node) (kube_node_status_capacity{resource="cpu"} - kube_node_status_allocatable{resource="cpu"}) >= 50
+----
+
+The Kubelet uses the system reserved CPU for its own operations and for running critical system services. For the node's health, it is important to ensure that system reserve CPU usage does not exceed the 50% threshold. Exceeding this limit could indicate heavy utilization or load on the Kubelet, which affects node stability and potentially the performance of the entire Kubernetes cluster. 
+
+If you observe any node is displayed in this metric, the Kubelet and the system overall are under heavy load. You can reduce overload on a particular node by balancing the load across other nodes in the cluster. Check other query metrics under the *Outliers*, *Average durations*, and *Number of operations* categories to gain further insights and take necessary corrective action.
```
Comment (adellape) [inline modules/nodes-dashboard-using-identify-critical-cpu-kubelet.adoc:18]: Wording at the beginning feels a bit awkward somehow. Maybe just:
```suggestion
If any node is displayed in this metric, the Kubelet and the system overall are under heavy load. You can reduce overload on a particular node by balancing the load across other nodes in the cluster. Check other query metrics under the *Outliers*, *Average durations*, and *Number of operations* categories to gain further insights and take necessary corrective action.
```

File: modules/nodes-dashboard-using-queries.adoc [hunk 10]
```diff
@@ -0,0 +1,16 @@
...
+
+:_content-type: PROCEDURE
+[id="nodes-dashboard-using-queries_{context}"]
+= Customizing dashboard queries
+
+You can customize the default queries used to build the node metrics dashboard.
+
+.Procedure
+
+. Choose a metric and click *Inspect* to drill into the data. On this page, you will see the metric in detail, including an expanded visualization of the results of the query, the Prometheus query used to analyze the data, and the data subset used in the query.
```
Comment (adellape) [inline modules/nodes-dashboard-using-queries.adoc:13]: IBM SG has warnings about use of "drill" or "drill down":
```suggestion
. Choose a metric and click *Inspect* to navigate into the data. On this page, you will see the metric in detail, including an expanded visualization of the results of the query, the Prometheus query used to analyze the data, and the data subset used in the query.
```

Comment (adellape) [inline modules/nodes-dashboard-using-queries.adoc:13]: Future tense:
```suggestion
. Choose a metric and click *Inspect* to drill into the data. This page displays the metric in detail, including an expanded visualization of the results of the query, the Prometheus query used to analyze the data, and the data subset used in the query.
```

File: modules/nodes-dashboard-using-queries.adoc [hunk 11]
```diff
@@ -0,0 +1,16 @@
...
+. Make any needed changes to the query parameters.
```
Comment (adellape) [inline modules/nodes-dashboard-using-queries.adoc:14]: "Needed" I believe is allowed (unlike "need to"), however a light suggestion for "required", if you like:
```suggestion
. Make any required changes to the query parameters. 
```

File: nodes/nodes-dashboard-using.adoc [hunk 12]
```diff
@@ -0,0 +1,50 @@
+:_content-type: ASSEMBLY
+[id="nodes-dashboard-using"]
+= The node metrics dashboard
```
Comment (adellape) [inline nodes/nodes-dashboard-using.adoc:3]:
```suggestion
= Node metrics dashboard
```

**Review by adellape — COMMENTED** [review]
Oops, hit Enter too soon.

**Review by adellape — APPROVED** [review]

**Pull request merged**